modules/overrides/util.go

```diff
@@ -0,0 +1,10 @@
+package overrides
+
+func doesNotContain(list []string, item string) bool {
```

Comment:
just not the output of this? https://pkg.go.dev/golang.org/x/exp/slices#Contains

Reply:
Woah, today I learned